src/FluentResults.Test/ResultWithValueTests.cs

```diff
@@ -277,6 +277,135 @@ public void ToResult_ToAnotherValueTypeWithOkResultAndConverter_ReturnSuccessRes
             result.IsSuccess.Should().BeTrue();
             result.Value.Should().Be(4);
         }
+        
+        [Fact]
```

Comment:
I didn't looked at the tests - let us discuss the other issues first.

Reply:
Ready for review, hopefully :)